properties/conftest.py

```diff
@@ -0,0 +1,5 @@
+from hypothesis import settings
+
+# Run for a while - arrays are a bigger search space than usual
+settings.register_profile("ci", deadline=None)
```

Comment:
:+1: on moving the settings profile to `conftest.py`, I was about to suggest that on the NetCDF PR.

I'd add `print_blob=True` to this, which prints more details to allow local reproduction of failures in CI.  It's a bit more verbose, so we leave it disabled by default, but if you ever need it you'll be very glad it exists.